spec/intl.html

```diff
@@ -1338,6 +1338,259 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateyear" type="abstract operation">
+          <h1>
+            CalendarDateYear (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to find the year for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonth" type="abstract operation">
+          <h1>
+            CalendarDateMonth (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to find the month for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonthcode" type="abstract operation">
+          <h1>
+            CalendarDateMonthCode (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to find the monthCode corresponding to the month for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateday" type="abstract operation">
+          <h1>
+            CalendarDateDay (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to find the day for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedayofweek" type="abstract operation">
+          <h1>
+            CalendarDateDayOfWeek (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It takes a date-like object _date_ and returns the day of the week in the calendar represented by _calendar_.</dd>
```

Comment:
From discussion with @ryzokuken, there are (at least) three possible ways to specify this: 
Option 1: All calendars must return 1=Monday, ..., 7=Sunday
Option 2: Every calendar can decide its own numbering
Option 3: Every calendar can decide its own numbering, but calendars SHOULD use 1=Monday, ..., 7=Sunday because that's what all currently supported ICU calendars do and deviating from this behavior may be unexpected. 

Note that the "first day of week" and "is this a weekday/weekend" is *not* a Temporal thing; it's part of localization, not calendar business logic. This is because for the Gregorian calendar, the first day of week differs across locales with US being Sunday and EU being Monday.

Reply:
Yeah I do think Option 1 is better here; we can have an additional function to convert a "day of week" into a "day number of week" or something.

We could call it ISODayOfWeek too.

I think having this be locale based would be quite confusing. Having _two_ APIs, one which is locale based, might make it clear though: whenever there are two similar APIs it's a good signal that you should check the docs :smile:

Reply:
Where did this discussion end up on the 402 side?  Note that adding new properties could be done in a follow-up proposal, but changing the semantics of existing properties would require a very high bar given how far we are in Stage 3.